src/differentiation/compute_jacobian_ad.jl

```diff
@@ -25,15 +27,21 @@ function ForwardColorJacCache(f::F,x,_chunksize = nothing;
         chunksize = _chunksize
     end
 
+    if tag === nothing
+        T = typeof(ForwardDiff.Tag(f,eltype(vec(x))))

```

Comment:
Why not set it to a custom tag here?
DiffEq is already using a few custom tags.

Reply:
This is for upstream packages to set package-specific tags.